hw/ip/alert_handler/data/alert_handler.hjson.tpl

```diff
@@ -2,14 +2,16 @@
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
 //
-# ALERT_HANDLER register template
-#
-# Parameter (given by Python tool)
-#  - n_alerts:    Number of alert sources
-#  - esc_cnt_dw:  Width of escalation counter
-#  - accu_cnt_dw: Width of accumulator
-#  - async_on:    Enables asynchronous sygnalling between specific alert RX/TX pairs
-#  - n_classes:   Number of supported classes (leave this at 4 at the moment)
+${autogen_banner}
+
+// ALERT_HANDLER register template
```

Comment:
Why are these // comments now?

Reply:
Fixed an inconsistency here (Hjson supports both styles, but `//` is used more predominantly).